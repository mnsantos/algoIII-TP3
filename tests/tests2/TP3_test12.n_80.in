80 1002
3 10
1 2
1 8
6 10
1 7
2 8
1 3
7 9
4 17
2 7
1 4
7 12
1 16
1 17
2 4
1 9
1 5
1 10
2 17
5 7
2 9
5 13
10 11
1 11
1 15
2 6
2 12
1 14
6 9
3 17
6 7
6 14
9 13
1 18
4 13
9 11
1 6
1 12
5 6
5 12
8 13
2 15
10 12
77 78
76 80
76 79
76 77
75 78
75 77
74 78
74 77
74 76
73 80
73 77
73 75
73 74
72 78
72 76
71 78
70 79
70 77
70 75
70 72
69 77
69 76
69 75
69 70
68 80
68 71
68 69
67 80
67 79
67 74
66 80
66 78
66 75
66 73
66 69
66 68
65 78
65 77
65 75
65 68
65 67
65 66
64 79
64 75
64 69
64 68
64 67
63 80
63 79
63 78
63 73
63 72
63 70
63 66
62 80
62 78
62 74
62 73
62 72
62 70
62 69
62 65
61 80
61 79
61 77
61 69
61 65
60 78
60 73
60 70
60 69
60 64
60 61
59 76
59 74
59 71
59 67
59 65
58 80
58 79
58 77
58 74
58 70
58 63
58 61
58 60
57 79
57 69
57 67
57 66
57 64
57 63
57 62
57 59
57 58
56 75
56 70
56 66
56 60
56 58
55 78
55 72
55 69
55 65
55 63
55 62
55 61
55 58
55 56
54 79
54 75
54 73
54 68
54 67
54 60
54 57
53 77
53 75
53 70
53 62
53 61
53 54
52 80
52 76
52 75
52 71
52 66
52 65
52 62
52 60
52 57
52 55
51 80
51 76
51 73
51 68
51 63
51 60
51 56
51 54
51 52
50 79
50 72
50 70
50 67
50 65
50 63
50 62
50 61
50 55
50 52
49 78
49 76
49 75
49 74
49 70
49 69
49 67
49 61
49 60
49 58
48 73
48 71
48 70
48 66
48 61
48 60
48 58
48 53
48 49
47 76
47 75
47 72
47 70
47 65
47 62
47 59
47 58
47 52
47 50
47 49
46 79
46 77
46 76
46 69
46 58
46 53
45 77
45 76
45 75
45 74
45 73
45 71
45 69
45 68
45 63
45 62
45 61
45 60
45 59
45 58
45 57
45 47
45 46
44 79
44 77
44 76
44 75
44 72
44 71
44 70
44 64
44 57
44 47
44 46
43 78
43 70
43 66
43 64
43 60
43 57
43 55
43 53
43 51
43 50
43 44
42 80
42 79
42 76
42 70
42 66
42 62
42 55
15 75
15 74
15 70
15 69
15 65
15 64
15 60
15 59
15 58
15 55
15 49
15 46
15 44
15 41
15 37
15 36
15 33
15 32
15 31
15 20
15 17
14 78
14 66
14 65
14 64
14 63
14 54
14 53
14 48
14 46
14 45
14 44
14 39
14 36
14 31
14 25
14 24
14 19
13 75
13 74
13 73
13 70
13 68
13 67
13 61
13 59
13 56
13 55
13 50
13 48
13 47
13 43
13 42
13 40
13 31
13 29
13 25
13 24
13 20
13 14
12 80
12 79
12 75
12 60
12 59
12 56
12 53
12 47
12 46
12 42
12 37
12 36
12 33
12 29
12 25
12 23
12 20
12 16
11 80
11 74
11 73
11 71
11 67
11 65
11 56
11 55
11 54
11 52
11 49
11 47
11 27
11 24
11 21
11 20
11 16
11 15
11 14
11 13
10 78
10 70
10 67
10 66
10 64
10 63
10 59
10 56
10 49
10 48
10 46
10 44
10 43
10 42
10 40
10 39
10 30
10 27
10 26
10 24
10 18
10 15
9 80
9 79
9 73
9 65
9 60
9 54
9 50
9 49
9 48
9 47
9 46
9 45
9 39
9 38
9 36
9 35
9 28
9 23
9 22
9 19
9 15
8 78
8 69
8 63
8 54
8 53
8 45
8 44
8 42
8 40
8 38
8 37
8 34
8 31
8 30
8 28
8 27
8 26
8 25
8 23
8 22
8 19
8 18
8 16
7 78
7 76
7 75
7 70
7 68
7 63
7 62
7 61
7 58
7 53
7 51
7 50
7 46
7 45
7 43
7 39
7 36
7 29
7 23
7 22
7 20
7 16
2 60
2 59
2 54
2 51
2 50
2 43
2 27
2 24
2 20
2 19
1 80
1 79
1 78
1 77
1 76
1 75
1 74
1 73
1 72
1 71
1 70
1 69
1 68
1 67
1 66
1 65
1 64
1 63
1 62
1 61
1 60
1 59
1 58
1 57
1 56
1 55
1 54
1 53
1 52
1 51
1 26
1 25
1 24
1 23
1 22
1 21
1 20
1 19
1 27
1 28
1 29
1 30
1 31
1 32
1 33
1 34
1 35
1 36
1 37
1 38
1 39
1 40
1 41
1 42
1 43
1 44
1 45
1 46
1 47
1 48
1 49
1 50
2 66
2 67
2 69
2 73
2 74
2 76
3 20
3 21
3 22
3 26
3 27
3 30
3 32
3 33
3 35
3 36
3 37
3 38
3 41
3 42
3 44
3 46
3 47
3 56
3 57
3 69
3 72
3 73
4 19
4 21
4 23
4 28
4 30
4 32
4 33
4 39
4 43
4 48
4 49
4 51
4 53
4 61
4 64
4 69
4 70
5 19
5 20
5 22
5 25
5 29
5 31
5 32
5 33
5 35
5 37
5 38
5 40
5 44
5 47
5 54
5 56
5 57
5 58
5 60
5 64
5 68
5 78
6 17
6 22
6 29
6 36
6 39
6 40
6 41
6 46
6 53
6 56
6 61
6 64
6 66
6 67
15 76
15 77
16 24
16 33
16 35
16 37
16 40
16 41
16 47
16 51
16 57
16 58
16 61
16 67
16 68
16 76
16 79
16 80
17 20
17 24
17 26
17 33
17 39
17 41
17 42
17 43
17 46
17 53
17 54
17 55
17 56
17 59
17 62
17 65
17 66
17 69
17 70
17 71
17 76
17 78
18 20
18 27
18 36
18 38
18 44
18 47
18 49
18 51
18 53
18 57
18 58
18 61
18 63
18 65
18 67
18 68
18 69
18 75
18 78
19 20
19 26
19 27
19 29
19 31
19 33
19 36
19 41
19 42
19 46
19 47
19 49
19 51
19 54
19 57
19 62
19 63
19 74
19 75
19 76
19 80
20 23
20 25
20 26
20 38
20 41
20 49
20 51
20 52
20 58
20 74
20 76
21 24
21 34
21 36
21 37
21 45
21 47
21 52
21 53
21 54
21 56
21 66
21 70
21 71
21 72
21 74
21 75
21 77
22 25
22 28
22 31
22 35
22 38
22 39
22 43
22 45
22 47
22 49
22 53
22 54
22 62
22 79
23 27
23 33
23 35
23 38
23 43
23 45
23 46
23 48
23 49
23 52
23 53
23 56
23 63
23 72
23 73
23 75
23 76
23 80
24 28
24 32
24 34
24 35
24 36
24 37
24 41
24 42
24 45
24 48
24 49
24 53
24 56
24 58
24 68
24 71
24 73
24 76
24 78
25 30
25 34
25 35
25 41
25 42
25 43
25 53
25 54
25 64
25 74
25 77
25 79
26 32
26 35
26 44
26 48
26 52
26 54
26 57
26 60
26 63
26 66
26 70
26 71
26 74
26 75
26 79
27 37
27 43
27 46
27 52
27 55
27 56
27 66
27 69
27 72
27 74
27 77
27 80
28 32
28 36
28 39
28 40
28 42
28 45
28 50
28 51
28 54
28 56
28 58
28 59
28 61
28 62
28 64
28 66
28 68
28 69
28 76
28 77
28 80
29 33
29 34
29 36
29 38
29 40
29 42
29 45
29 47
29 49
29 53
29 55
29 62
29 68
29 71
29 80
30 37
30 40
30 48
30 52
30 57
30 59
30 60
30 77
30 79
30 80
31 42
31 46
31 48
31 49
31 51
31 55
31 56
31 57
31 58
31 62
31 66
31 68
31 70
31 71
31 77
31 80
32 33
32 36
32 41
32 44
32 49
32 53
32 54
32 57
32 59
32 67
32 77
33 40
33 44
33 51
33 59
33 60
33 62
33 64
33 72
33 73
33 74
33 75
33 76
33 77
33 79
33 80
34 36
34 37
34 45
34 46
34 50
34 51
34 55
34 58
34 60
34 63
34 64
34 67
34 70
34 74
34 76
34 78
34 80
35 37
35 40
35 41
35 55
35 56
35 58
35 60
35 62
35 73
35 77
35 78
35 79
36 39
36 41
36 48
36 59
36 60
36 61
36 62
36 70
36 71
36 77
36 80
37 38
37 48
37 51
37 53
37 55
37 57
37 58
37 60
37 62
37 66
37 67
37 69
38 42
38 44
38 46
38 47
38 49
38 50
38 52
38 55
38 58
38 59
38 62
38 64
38 69
38 73
38 74
38 76
38 78
39 47
39 51
39 55
39 59
39 61
39 75
40 46
40 47
40 50
40 58
40 62
40 63
40 64
40 70
40 74
40 75
40 76
40 79
41 43
41 44
41 45
41 49
41 56
41 60
41 65
41 71
41 73
41 80
42 53
3 14
1 13
3 5
0
