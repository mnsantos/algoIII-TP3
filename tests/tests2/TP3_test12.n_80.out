197 6 1 15 49 58 60 70
